spec/system/organization_system_spec.rb

```diff
@@ -50,6 +50,17 @@
         expect(page).to have_content("Show Year-to-date values on distribution printout?")
         expect(page).to have_content("Logo")
         expect(page).to have_content("Use One step Partner invite and approve process?")
+
+        expect(page).to_not have_content("Custom Request Units")
+      end
+
+      it "Shows custom request units when enabled", :aggregate_failures do
```

Comment:
Can this be converted into a request spec?

Reply:
Looks like the system spec is still here :)

Reply:
OK updated now -- I moved almost all of the system specs over to the request spec, leaving only two that have pop-up interactions.

Reply:
Note that several of these tests I moved basically as-is though they are flawed -- like they are looking for the word "Yes" on the page as their success, but all the other fields could have "Yes" visible. Unchanged validation we should address separately.